### PR TITLE
Pull Request decorator improvemnet

### PR DESCRIPTION
**Pull request by TsuyoshiUshio**

### CODE REVIEW

File: VulnerableApplication/Controllers/HomeController.cs [hunk 1]
```diff
@@ -30,5 +30,8 @@ public ActionResult Contact()
         public string Hello() {
         return "this is not on the coding rules.";
         }
+        
+        private static string ChangePassword = "PassWord917231";
```
Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: **CODE_SMELL**
> 4 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWtnasPNBeEiCS_yR8GX&pullRequest=10&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: **CODE_SMELL**
> 10 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWtnasPNBeEiCS_yR8GY&pullRequest=10&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: **CODE_SMELL**
> Add or update the header of this file.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWtnasPNBeEiCS_yR8GZ&pullRequest=10&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: **CODE_SMELL**
> Remove the unused private field 'ChangePassword'.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWtnasPNBeEiCS_yR8Ga&pullRequest=10&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: **VULNERABILITY**
> 'password' detected in this expression, review this potentially hardcoded credential.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWtnasPNBeEiCS_yR8Gb&pullRequest=10&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: **CODE_SMELL**
> The field 'HomeController.ChangePassword' is assigned but its value is never used
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWtnasPNBeEiCS_yR8Gd&pullRequest=10&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Controllers/HomeController.cs:34]: **CODE_SMELL**
> Remove this method and declare a constant for this value.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AWtnasPNBeEiCS_yR8Gc&pullRequest=10&resolved=false)

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: /workitem

Comment (TsuyoshiUshio) [reply VulnerableApplication/Controllers/HomeController.cs:34]: WorkItem 774 Created see [workItem](https://dev.azure.com/csedevops/ea38b5b8-2b4f-4721-8514-fe765aa4a23a/_workitems/edit/774).
